collectors/pod.go

```diff
@@ -137,6 +137,18 @@ var (
 		"The limit on memory to be used by a container in bytes.",
 		[]string{"namespace", "pod", "container", "node"}, nil,
 	)
+
+	descPodContainerResourceRequestsGPU = prometheus.NewDesc(
+		"kube_pod_container_resource_requests_gpu",
+		"The number of requested gpu by a container.",
```

Comment:
ditto.

`The number of requested gpu by a container.` -> `The number of requested gpu cores by a container.`